fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java

```diff
@@ -145,6 +146,31 @@ protected static HttpPut putDSMethod(final String pid, final String ds, final St
         return put;
     }
 
+    protected static HttpPut putDSFileMethod(final String pid, final String ds, final File content)
```

Comment:
Why are any of these new methods in `AbstractResourceIT` still here? None of them are being used anymore.